### DISCUSSION TITLE
Dongle communication error (ackStatus==None)

### DISCUSSION THREAD

**Discussion by Cbabe** [bitcraze/discussions#828] [Q&A]
Hello,

I'm trying to connect to a Crazyfly Bolt over the radio through the Crazyfly VM in the GUI interface. The Crazyfly Bolt radio is discovered when I scan for it, but when I press the connect button the program hangs and freezes on connecting. The terminal output from the GUI reads "ackStatus==None". I've tried cold-booting the Crazyfly Bolt over the radio and successfully flashed the Bolt firmware, but still no successful connection to the Bolt over the GUI interface. I have also ran the "update all projects" executable which completed successfully. Any suggestions to get the connection working in the GUI?

**Comment by ataffanel** [comment]
Hi, What operating system are you using and are you using the virtual machine or running the client naively on your computer?

**Reply by Cbabe** [reply]
I'm running the virtual machine in virtualbox on a Windows 10 OS.

**Reply by ataffanel** [reply]
Is it possible for you to try to run natively by installing Python on Windows? (Python 3.8 will work best currently).

One other thing to try would be to change the USB setting in the virtual machine configuration to, for example, USB-3. This might work better.

Overall, USB to the virtual machine has always been a bit slow. The ackStatus==None is a symptom of that but normally do not cause any problems.

**Reply by Cbabe** [reply]
I'll try running it natively with python in windows. The settings in virtual box I believe are USB-3. Typically I get the 
ackstatus==0 couple of times before the radio connection is disconnect.
